src/features/web-compat.js

```diff
@@ -283,6 +292,68 @@ export default class WebCompat extends ContentFeature {
         window.navigator.permissions = permissions
     }
 
+    /**
+     * Fixes screen lock/unlock APIs for Android WebView.
+     */
+    screenLockFix () {
+        const validOrientations = [
+            'any',
+            'natural',
+            'landscape',
+            'portrait',
+            'portrait-primary',
+            'portrait-secondary',
+            'landscape-primary',
+            'landscape-secondary',
+            'unsupported'
+        ]
+
+        this.wrapProperty(globalThis.ScreenOrientation.prototype, 'lock', {
+            value: async (requestedOrientation) => {
+                this.addDebugFlag()
```

Comment:
One to validate, but I think we should be taking care of this now: https://github.com/duckduckgo/content-scope-scripts/blob/012b3925944a3cd5338aef51a42736448d254937/src/content-feature.js#L287

Reply:
Tested - looks like it's indeed not needed. Fixed, thanks!